### PR TITLE
Swapped LayerNorm for BatchNorm, fixed e3nn dropout, and made growthr…

### PR DESCRIPTION
**Pull request by jmcguigan10**
…ate rescaling optional(seemed to help with convergence.)

### CODE REVIEW

File: model_training/ml_neuralnet.py [hunk 1]
```diff
@@ -152,6 +152,11 @@ def __init__(self, parms):
 
         # store input arguments
         self.NF = parms["NF"]
+        # allow legacy configs that used do_layernorm
```
Comment (srichers) [inline model_training/ml_neuralnet.py]: I would favor code simplicity over legacy support. Could you just remove support for layernorm?

File: model_training/ml_neuralnet.py [hunk 2]
```diff
@@ -152,6 +152,11 @@ def __init__(self, parms):
...
+        do_batchnorm = parms.get("do_batchnorm", parms.get("do_layernorm", False))
+        # optionally scale predicted growthrate back to physical units
+        # default False because growthrate labels are already normalized in the dataset
+        self.scale_growthrate_by_ntot = parms.get("scale_growthrate_by_ntot", False)
```
Comment (srichers) [inline model_training/ml_neuralnet.py]: I think this should also just be baked in without an option. The growthrate should absolutely be scaled by ntot. The issue is that even if the training datasets are labeled, the network is set up to do predictions on un-normalized inputs. We could expect the user to scale the growhtrate, but I think it is simplest to just do it in-house, so the outputs are all in consistent units.

File: .DS_Store
Comment (srichers) [inline .DS_Store:1]: Why is this file in the repository?

**Review by srichers — CHANGES_REQUESTED** [review]
This is great, thanks! A couple of small suggestions - let me know what you think.

File: .DS_Store
Comment (jmcguigan10) [reply .DS_Store:1]: I didn't mean to push that my bad.

**Review by srichers — APPROVED** [review]
